assets/components/LogViewer/SimpleLogItem.vue

```diff
@@ -28,7 +28,14 @@ defineProps<{
 }>();
 
 const colorize = (value: string) => ansiConvertor.toHtml(value);
-const urlPattern = /(https?:\/\/[^\s]+)/g;
+const urlPattern =
+  /https?:\/\/(?:www\.)?[-a-zA-Z0-9@:%._+~#=]{1,256}\.[a-zA-Z0-9()]{1,6}\b[-a-zA-Z0-9()@:%_+.~#?&\/=]*/g;
```

Comment:
This is based on [this](https://stackoverflow.com/a/3809435/965834) and slightly improved (Stack Overflow isn't always a reliable source but when an answer has been maintainted/upvoted for so long, it's probably a good sign).

Reply:
Nice find.